README.md

```diff
@@ -261,6 +261,13 @@ policies:
   regex_pattern: "\d"
 ```
 
+- `DeleteByRegexpPath` - delete artifacts whose path matches the specified regexp
```

Comment:
May be `FilterByRegexpPath`? :thinking: 

Because it actually doesn't remove any, more like Filters like here
https://github.com/devopshq/artifactory-cleanup?tab=readme-ov-file#filters

Reply:
agreed. renamed